data/styles/clear/include/Basemap_label.mapcss

```diff
@@ -349,10 +352,10 @@ node|z9[place=town][population>=20000]
 node|z9[place=town][population>=20000]::int_name
 {font-size: 9;}
 
-node|z10[place=town][population<20000]
-{font-size: 10;text-color: @city_label;text-halo-opacity: 0.7;}
-node|z10[place=town][population<20000]::int_name
-{font-size: 9;text-color: @city_label;text-halo-opacity: 0.7;}
+node|z10[place=town][population>=0]
```

Comment:
Used simple node|z10[place=town] above.
Is there any difference with [population>=0] ? I'm not sure for features _with and without_ stored population ...

Reply:
I used the same approach used for cities in a section above and it seems to be working well.
I'll post more screenshots in a moment.